mc.tests/x86-mov.exp

```diff
@@ -93,11 +93,9 @@ set suite(x86) {
 
     {0x66,0x8c,0xe0} {movw %fs, %ax} {EAX := high:16\[EAX\].FS}
     {0x8c,0xe0} {movl %fs, %eax} {EAX := pad:32\[FS\]}
-    {0x8c,0x20} {movl %fs, (%eax)} {mem := mem with \[EAX, el\]:u16 <- FS}
```

Comment:
for clarification: we're removing this test case as it is actually a demonstration of a bug in LLVM version prior to 6. The `\x8c\x20` instruction is actually `movw` not `movl`. Our lifters, both legacy (that doesn't depend on LLVM) and modern (that uses LLVM) representation lift this instruction correctly for all version of LLVM as they only look at the size of the argument (that is always 16 bits in case of a segment register). So this test case was only checking for an (incorrect) assembly output.